wikipedia-dark.user.css

```diff
@@ -427,6 +427,11 @@ domain("wiki.rpcs3.net") {
   .oo-ui-buttonElement-frameless.oo-ui-widget-enabled.oo-ui-flaggedElement-destructive > .oo-ui-buttonElement-button:hover {
     color: #ff4242 !important;
   }
+  /*** Quotes ***/
+  .ts-Начало_цитаты-quote {
```

Comment:
Hi @TotalCaesar659!

Thanks for reporting this issue and for your contribution!

It looks like a more generic `blockquote` can be used instead of `.ts-Начало_цитаты-quote` to fix that style. Please update this PR and I'll merge it. Thanks!

Reply:
Sorry, I had a doubt about it, I'm not so good in changing HTML styles. Fixed.